doc/config-schemas/setDeleteAlarmPartitionZone.md

```diff
@@ -0,0 +1,46 @@
+# Deletes a zone from a partition
+
+## Descripcion
+
+Remueve una zona de una particion
+
+## Request Message
+
+```json
+{
+    ...
+    "data": {
+        "deviceData": {...},
+        "configKey": "setDeleteAlarmPartitionZone",
+        "value":{
+            "number":1,
+            "systemId": "xxx-xxxx",
+            "zoneNumber":14
```

Comment:
check this comment
https://github.com/Netsocs-Team/driver.sdk_go/pull/15#issuecomment-2031992105